src/NzbDrone.Core/Indexers/Exceptions/IndexerException.cs

```diff
@@ -18,6 +18,16 @@ public IndexerException(IndexerResponse response, string message)
             _indexerResponse = response;
         }
 
+        public IndexerException(string message)
+            : base(message)
+        {
+        }
```

Comment:
Allowing initialization without setting response may result in unwanted behavior downstream in history processing. HttIndexerBase query error handling expects a response to be present when handling IndexerExceptions

Reply:
Added

Reply:
hmm actually needed to add that as the existing

```chsarp
        public IndexerException(IndexerResponse response, string message)
            : base(message)
        {
            _indexerResponse = response;
        }
```
was causing VS to not like it

thoughts?

Reply:
builds DOA pending figuring out how to handle these; 

we need to not require the response to be ingested or change the cardigann exceptions to include the responses